src/context/PendingNotificationsContext.tsx

```diff
@@ -48,8 +48,7 @@ export const PendingNotificationsProvider = ({ children }) => {
       const notificationId = notification.request.content.data.uniqueId
       const notAddedYet =
         pendingNotifications.findIndex(
-          notification =>
-            notification.request.content.data.uniqueId === notificationId
+          item => item.request.content.data.uniqueId === notificationId
```

Comment:
why this rename?

Reply:
Notification is already defined. I'm happy to update to a better name -- I just couldn't think of one!

Reply:
generally speaking shadowing variables is not a linting error, is it in this case? if so, fine with this change

Reply:
Yes, it looks like these are showing as linting warnings in the link attached to the issue.

Reply:
okay fair enough